lib/ssl/src/tls_server_connection_1_3.erl

```diff
@@ -592,8 +592,8 @@ send_hello_flight({start_handshake, PSK0},
         {Ref, #alert{} = Alert} ->
             Alert;
         error:Reason:ST ->
-            ?SSL_LOG(debug, crypto_error, [{reason, Reason}, {stacktrace, ST}]),
-            ?ALERT_REC(?ILLEGAL_PARAMETER, illegal_parameter_to_compute_key)
+            ?SSL_LOG(info, crypto_error, [{reason, Reason}, {stacktrace, ST}]),
```

Comment:
Would this be worth fixing on maint as it is a bug?

Reply:
Fixed moved the commit to separate branch.